src/globus_sdk/services/compute/client.py

```diff
@@ -3,9 +3,7 @@
 import logging
 import typing as t
 
-from globus_sdk import GlobusHTTPResponse, client
-from globus_sdk._missing import MISSING, MissingType
-from globus_sdk._remarshal import strseq_listify
+from globus_sdk import MISSING, GlobusHTTPResponse, MissingType, client, utils
```

Comment:
This is the line which causes the failure, since there is no longer a module named `globus_sdk.utils`.

I believe the removed lines are actually the desirable ones here.

Reply:
@sirosen this should be resolved now. I've inverted the changes there.